authentikos/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM alpine:3.10.3
```

Comment:
A distroless image seems more appropriate here, why do we need alpine?

Reply:
> A distroless image seems more appropriate here, why do we need alpine?

What is a "distroless" image?  I went with `alpine` because it is only 5MB.

Reply:
https://github.com/GoogleContainerTools/distroless

Reply:
👍 will check that out!

Reply:
👍 will check that out!

Reply:
I am now using a *distroless* docker image!